x-pack/legacy/plugins/apm/public/components/app/Main/ProvideBreadcrumbs.tsx

```diff
@@ -14,7 +14,7 @@ import {
 } from 'react-router-dom';
 import { RouteName } from './route_config/route_names';
 
-type LocationMatch = Pick<
+export type LocationMatch = Pick<
```

Comment:
I don't think we necessarily need to export this anymore if the other feedback is addressed